packages/lib-utils/src/app/redux/reducers/k8s/index.ts

```diff
@@ -1,15 +1,271 @@
+import { consoleLogger } from '@monorepo/common';
 import { Map as ImmutableMap, fromJS } from 'immutable';
-import type { ActionType as Action } from 'typesafe-actions';
-import type { K8sModelCommon } from '../../../../types/k8s';
+import * as _ from 'lodash-es';
+import { getReferenceForModel, getNamespacedResources, allModels } from '../../../../k8s/k8s-utils';
+import type { K8sModelCommon, K8sResourceCommon } from '../../../../types/k8s';
 import type { K8sState } from '../../../../types/redux';
+import type { K8sAction } from '../../actions/k8s';
+import { ActionType } from '../../actions/k8s';
+import { getK8sDataById } from './selector';
 
-// TODO remove when actions/reducers are migrated
-const emptyK8sState: K8sState = fromJS({
-  RESOURCES: { inFlight: false, models: ImmutableMap<string, K8sModelCommon>() },
-});
+const getQN: (obj: K8sResourceCommon) => string = (obj: K8sResourceCommon) => {
```

Comment:
```suggestion
const getQN: (obj: K8sResourceCommon) => string = (obj) => {
```

Don't need to double type.